internal/resourcepolicies/volume_types_conditions.go

```diff
@@ -0,0 +1,239 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resourcepolicies
+
+import (
+	corev1api "k8s.io/api/core/v1"
+)
+
+const (
+	GCEPersistentDisk    = "gcePersistentDisk"
+	AWSElasticBlockStore = "awsElasticBlockStore"
+	HostPath             = "hostPath"
+	Glusterfs            = "glusterfs"
+	NFS                  = "nfs"
+	RBD                  = "rbd"
+	ISCSI                = "iscsi"
+	Cinder               = "cinder"
+	CephFS               = "cephfs"
+	FC                   = "fc"
+	Flocker              = "flocker"
+	FlexVolume           = "flexVolume"
+	AzureFile            = "azureFile"
+	VsphereVolume        = "vsphereVolume"
+	Quobyte              = "quobyte"
+	AzureDisk            = "azureDisk"
+	PhotonPersistentDisk = "photonPersistentDisk"
+	PortworxVolume       = "portworxVolume"
+	ScaleIO              = "scaleIO"
+	Local                = "local"
+	StorageOS            = "storageOS"
+	CSI                  = "csi"
+	EmptyDir             = "emptyDir"
+	GitRepo              = "gitRepo"
+	Secret               = "secret"
+	DownwardAPI          = "downwardAPI"
+	ConfigMap            = "configMap"
+	Projected            = "projected"
+	Ephemeral            = "ephemeral"
+)
```

Comment:
2 suggestions:
- would be interesting to transform this in a type, like `supportedVolumes`, and change the return values of `getVolumeTypeFromPV` and `getVolumeTypeFromVolume` functions to it?
- would be interesting to sort the volumes in alphabetical order? (and also in the `getVolumeTypeFromPV` and `getVolumeTypeFromVolume` functions bodies)

Reply:
Thanks for your suggestions, I've updated